wolftpm/tpm2_wrap.h

```diff
@@ -1085,6 +1085,32 @@ WOLFTPM_API int wolfTPM2_ImportEccPrivateKey(WOLFTPM2_DEV* dev,
     \sa wolfTPM2_LoadEccPrivateKey_ex
     \sa wolfTPM2_LoadPrivateKey
 */
+/*!
```

Comment:
Please add new line above the doxygen section for each new one added. This allows for easier readability.